homeassistant/components/ping/binary_sensor.py

```diff
@@ -113,11 +111,11 @@ def is_on(self) -> bool:
     @property
     def extra_state_attributes(self) -> dict[str, Any] | None:
         """Return the state attributes of the ICMP checo request."""
-        if self.coordinator.data.data is None:
+        if len(self.coordinator.data.data) == 0:
```

Comment:
The attributes should always be there

Reply:
Ok, just kept the prev. behavior but will remove this